docs/src/input.md

```diff
@@ -9,21 +9,50 @@ title: "Input"
 
 This is the easiest way to fill out the form fields. It focuses the element and triggers an `input` event with the entered text. It works for `<input>`, `<textarea>`, `[contenteditable]` and `<label>` associated with an input or textarea.
 
-```js
-// Text input
-await page.fill('#name', 'Peter');
+```js js-flavor=ts RUNNABLE
+import { test, expect } from '@playwright/test';
 
-// Date input
-await page.fill('#date', '2020-02-02');
+test('form examples', async ({ page }) => {
+  await page.goto('https://rwoll.github.io/gh-pages-examples/');
```

Comment:
All these examples will point at official domain; this site is currently a placeholder.